play.pokemonshowdown.com/src/client-main.ts

```diff
@@ -261,6 +263,9 @@ export interface Team {
 	/** The icon cache must be cleared (to `null`) whenever `packedTeam` is modified */
 	iconCache: preact.ComponentChildren;
 	key: string;
+	teamid?: number;
```

Comment:
I guess this is the ID from the teams database? Plausibly these should be documented.